src/bionics.cpp

```diff
@@ -430,8 +433,38 @@ bool Character::activate_bionic( int b, bool eff_only )
             }
         }
 
-        const size_t win_h = std::min( static_cast<size_t>( TERMY ), bad.size() + good.size() + 2 );
-        const int win_w = 46;
+        const std::map<vitamin_id, vitamin> &vit_all = vitamin::all();
+        for( const auto &v : vit_all ) {
+            if( !v.second.has_flag( "NO_DISPLAY" ) && !v.second.has_flag( "NOT_IN_BLOOD" ) ) {
```

Comment:
Please document the new flag and the additional behavior of the old one.

Reply:
Maybe we shouldn't check NO_DISPLAY here at all. If vitamin is not showing in comestibles UI doesn't necessary mean it shouldn't be displayed by Blood Analyzer.

Reply:
I think there's definitely use for a flag that means that the vitamin is never displayed - perhaps the food one should have a separate flag?

Reply:
I wanted to use NOT_IN_BLOOD flag to tell Blood Analyzer to ignore that vitamin... Maybe should change it to BLOOD_ANALYZER to do the opposite - only show vitamins _with_ this flag.